hexrd/wppf/WPPF.py

```diff
@@ -367,7 +376,7 @@ def initialize_Icalc(self):
 
         if self.intensity_init is None:
             if self.spectrum_expt._y.max() > 0:
-                n10 = np.floor(np.log10(self.spectrum_expt._y.max())) - 2
+                n10 = np.floor(np.log10(self.spectrum_expt._y.max())) - 1
```

Comment:
Maybe the `-2` was a typo? :slightly_smiling_face:

Reply:
-2 is intentional! This bit is trying to initialize the background and the starting assumption is that the background is $10^2$ times lower than the peak signal. This just seems to work better than $10^1$.